### ISSUE TITLE
tensorflow pb file to mxnet, predictions differed

### ISSUE THREAD

**Issue by jayle19930918** [microsoft/MMdnn#749]
Platform (like ubuntu 16.04/win10): 
win10

Python version: 
3.6.9

Source framework with version (like Tensorflow 1.4.1 with GPU): 
Tensorflow 1.8.0 with GPU

Destination framework with version (like CNTK 2.3 with GPU): 
mxnet 1.5.0 with GPU

Pre-trained model path (webpath or webdisk path):

Running scripts:
mmconvert -sf tensorflow -iw firenet_new.pb --inNodeName InputData/X --inputShape 224,224,3 --dstNodeName FullyConnected_2/Softmax -df mxnet -om mxnet_firenet -dw mxnet_firenet-0000.params

The tensorflow .pb file was converted to mxnet model successfully using the above scripts. However, the predictions of the converted model were [nan, nan]. I printed the layer name of firenet.pb and the converted model. The converted model did not have MaxPooling layer and LRN layer compared to the original model. I want to know if there are some problems about the above scripts.
